internal/cri/server/container_create.go

```diff
@@ -161,18 +161,24 @@ func (c *criService) CreateContainer(ctx context.Context, r *runtime.CreateConta
 		return nil, fmt.Errorf("failed to query sandbox platform: %w", err)
 	}
 
+	ociRuntime, err := c.config.GetSandboxRuntime(sandboxConfig, sandbox.Metadata.RuntimeHandler)
+	if err != nil {
+		return nil, fmt.Errorf("failed to get sandbox runtime: %w", err)
+	}
+
 	var volumeMounts []*runtime.Mount
 	if !c.config.IgnoreImageDefinedVolumes {
+		err = c.mutateMounts(ctx, config.GetMounts(), sandboxID, c.RuntimeSnapshotter(ctx, ociRuntime), platform)
+		if err != nil {
+			return nil, fmt.Errorf("failed to mount image volume: %w", err)
+		}
+
 		// Create container image volumes mounts.
```

Comment:
create a list of image volume mounts from the image spec that are not also already in the runtime config volume list
```suggestion
		// create a list of image volume mounts from the image spec that are not also already in the runtime config volume list